linkace_cli/cli/links.py

```diff
@@ -157,7 +157,6 @@ def update(
             link[key] = val
 
     link['tags'] = ", ".join([tag['name'] for tag in link['tags']])
-    link['lists'] = ", ".join([alist['name'] for alist in link['lists']])
 
     link = {key: link[key] for key in provided_info.keys()}
     if interactive:
```

Comment:
Adding the tags and list processing inside the interactive portion of the code.

```suggestion
    if interactive:
    	link['tags'] = ", ".join([tag['name'] for tag in link['tags']])
        link['lists'] = ", ".join([alist['name'] for alist in link['lists']])
```

Thanks!